lib/gui/src/ide.rs

```diff
@@ -0,0 +1,23 @@
+//! This module defines the entrypoint function for IDE.
+
+use wasm_bindgen::prelude::*;
+
+use basegl::system::web;
+use ide::entry_point::entry_point;
+
+/// IDE startup function.
+#[wasm_bindgen]
+#[allow(dead_code)]
+pub fn run_example_ide() {
+    web::forward_panic_hook_to_console();
+    web::set_stdout();
+    basegl_core_msdf_sys::run_once_initialized(|| {
+        web::get_element_by_id("loader").map(|t| {
```

Comment:
"loader" is a literal that I don't know where it comes from or what against it needs to be synchronized.
Also, this piece of code is duplicated with shapes example, I'd consider refactoring it to some function like `remove_loader`.

Reply:
@mwu-tow this is a temporary fix suggested by @wdanilo

Reply:
It'd be good to have this information in the comment.
(also, fix for what issue? if some future me reaches this code, how can I know it can be removed?)